gradle/libs.versions.toml

```diff
@@ -0,0 +1,49 @@
+[versions]
+kotlin = "1.9.21"
+kotlinx-metadata = "0.7.0"
+asm = "9.6"
+slf4j = "1.8.0-alpha2"
+junit = "4.12"
+maven = "3.5.3"
+node-gradle = "3.1.1"
+node = "8.11.1"
+npm = "5.7.1"
+rhino = "1.7.10"
+
+[libraries]
+
+# Kotlin dependencies
+kotlin-gradlePlugin = { module = "org.jetbrains.kotlin:kotlin-gradle-plugin", version.ref = "kotlin" }
+kotlin-atomicfu = { module = "org.jetbrains.kotlin:atomicfu", version.ref = "kotlin" }
+kotlin-stdlib = { module = "org.jetbrains.kotlin:kotlin-stdlib", version.ref = "kotlin" }
+kotlin-stdlibJdk8 = { group = "org.jetbrains.kotlin", name = "kotlin-stdlib-jdk8", version.ref = "kotlin" }
+kotlin-test = { group = "org.jetbrains.kotlin", name = "kotlin-test", version.ref = "kotlin" }
+kotlin-testJunit = { group = "org.jetbrains.kotlin", name = "kotlin-test-junit", version.ref = "kotlin" }
+kotlin-scriptRuntime = { group = "org.jetbrains.kotlin", name = "kotlin-script-runtime", version.ref = "kotlin" }
+kotlinx-metadataJvm = { group = "org.jetbrains.kotlinx", name = "kotlinx-metadata-jvm", version.ref = "kotlinx-metadata" }
+
+# ASM dependencies
+asm = { group = "org.ow2.asm", name = "asm", version.ref = "asm" }
+asm-commons = { group = "org.ow2.asm", name = "asm-commons", version.ref = "asm" }
+asm-tree = { group = "org.ow2.asm", name = "asm-tree", version.ref = "asm" }
+asm-util = { group = "org.ow2.asm", name = "asm-util", version.ref = "asm" }
+
+# Logging dependencies
+slf4j-api = { group = "org.slf4j", name = "slf4j-api", version.ref = "slf4j" }
+slf4j-simple = { group = "org.slf4j", name = "slf4j-simple", version.ref = "slf4j" }
+
+# Testing depedencies
+junit-junit = { group = "junit", name = "junit", version.ref = "junit" }
+
+# Maven dependencies
+maven-core = { group = "org.apache.maven", name = "maven-core", version.ref = "maven" }
+maven-pluginApi = { group = "org.apache.maven", name = "maven-plugin-api", version.ref = "maven" }
+maven-pluginAnnotations = { group = "org.apache.maven.plugin-tools", name = "maven-plugin-annotations", version = "3.5" }
```

Comment:
Nitpick:

I think it's a good idea to have all versions defined under `[versions]`. It helps with consistency, and can help minimise changes if, in the future, a version does end up being re-used.

So, I'd suggest changing `version = "3.5"` to `version.ref = "maven-pluginTools"`, and adding a version.

(Just for reference, Dokka has the same `maven-plugin-annotations` dependency: https://github.com/Kotlin/dokka/blob/4f5a650d6ae8f4f94c34ae4bedf72e7c660aecbe/gradle/libs.versions.toml#L128)

Reply:
fixed